protocol/message_persistence.go

```diff
@@ -1354,6 +1356,36 @@ func (db sqlitePersistence) SetHideOnMessage(id string) error {
 	return err
 }
 
+func (db sqlitePersistence) DeleteMessagesByCommunityID(id string) error {
+	return db.deleteMessagesByCommunityID(id, nil)
+}
+
+func (db sqlitePersistence) deleteMessagesByCommunityID(id string, tx *sql.Tx) (err error) {
+	if tx == nil {
```

Comment:
Why passing transaction from outside? Is it recycled/batched somewhere?

Reply:
This is a leftover. Good catch!